deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml

```diff
@@ -23,32 +23,11 @@ rules:
   verbs:
   - get
   - watch
-# ACS team can delete pods
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - delete
-# ACS team can run commands in pods
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - create
-# ACS team can restart deployments/statefulsets/deploymentconfigs
-- apiGroups:
-  - apps
-  resources:
-  - statefulsets
-  - deployments
-  verbs:
-  - patch
+  - list
 # ACS team can scale deployments
 - apiGroups:
   - apps
   resources:
   - deployments/scale
   verbs:
-  - patch
\ No newline at end of file
+  - get
```

Comment:
Requirements state "scale deployments" but this PR removes `patch` permission.  Is it granted somewhere else?

Reply:
changed to deployments